target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -225,6 +225,12 @@ ipq40xx_setup_macs()
 	pakedge,wr-1)
 		wan_mac=$(macaddr_add $(get_mac_label) 1)
 		;;
+	tel,x1pro)
+		label_mac=$(mtd_get_mac_binary "0:ART" 0x1006)
+		wan_mac=$(macaddr_add "$label_mac" 1)
+		lan_mac=$(macaddr_add "$label_mac" 2)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
```

Comment:
This doesn't go here, function for interface setting is above the mac setting one.